examples/prediction_upload/video_predictions.ipynb

```diff
@@ -64,18 +64,7 @@
         "!pip install -q 'labelbox[data]'"
```

Comment:
<p>Same comment as before, let's pass in a list of global_keys by wrapping <code>global_key</code> in a list: <code>[global_key]</code> .</p><p>Also, we should update the comment next to it </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>